base.php

```diff
@@ -114,3 +115,11 @@
 	</body>
 				<?php } ?>
 </html>
+<?php } elseif ( $is_sa_estimate ) {
```

Comment:
There's 100's of themes and plugins out there. Is the plan to continue adding additional `elseif` statements for each one?

Reply:
This logic has been moved out of the template file.